node-authorizer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM fedora:27
```

Comment:
nitpick: I believe the preferred base image is debian, e.g. [`k8s.gcr.io/debian-base-amd64:0.3`](https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/debian-base-amd64?gcrImageListsize=50)